workflow-to-js-transformer/src/main/java/com/github/cafdataprocessing/workflow/transform/FullWorkflowRetriever.java

```diff
@@ -178,4 +240,61 @@ private FullAction buildFullAction(
         }
         return new FullAction(existingAction, actionConditions);
     }
+
+    private long getWorkflowId(final String projectId, final String workflowName)
+        throws ApiException, InvalidWorkflowSpecificationException
+    {
+        final WorkflowsApi workflowsApi = apisProvider.getWorkflowsApi();
+        final ExistingWorkflows existingWorkflows = workflowsApi.getWorkflows(projectId, 1, 100);
```

Comment:
Presumably this code breaks down if there are more than 100 workflows.

I'm OK with that as I'd expect a much smaller number.